### PR TITLE
More asteroids honk

### PR DESCRIPTION
**Pull request by GoatMonarch**
## About The Pull Request
Hello this is a mega pack of new asteroids!
We went from 7 asteroids to 30 diferent asteroids. With varrying gimmicks.
I also added a pack of cheesy honkers beneath the AI core on the aetherwhisp ruin, since the original map always had that.

- [*] Make all the maps, map merge them
- [*] Increase ruin spawn chance, since we now have more than seven ruins

**List of what the maps are:**
Cat surgeon asteroid
Asteroid with inside water, has a slug potion inside.
Asteroid with a "melted" down stormdrive. Stormdrive is actually a drink container.
Crashed cargo shuttle with a bunch of crates. Two weak augmentation parts hidden behind a bunch of maint loot.
Asteroid with Ratvar just chilling.
Asteroid with lavaland turf, and three gholiats.
Asteroid that is literally a square.
Asteroid simmilar to the asteroid shuttle.
Asteroid with plasma pockets inside **fastmost go brr**
Xenobio lab, but all of the slimes are loose! No consoles to handle it either.
A space carp dojo. Included are fake weapons that deals 30 stam damage on hit, carp suits. And some four (one eliete) hostile carp
Asteroid with several bio-domes around.
Vault asteroid. Rare material (5 ore of bananium), but you need to open it the secured vault with a stetoscope. (Legit never seen nobody do this.)
Ship crashed into a rock, has an intact teleporter aboard.
A small shrine with a hilberts hotel
Rock with an iglo on it. Two skimoes, all terain viechle.
Asteroid stuffed with geese.
Asteroid with a little open space of a "brick" 
Space ikea. Furniture, some raw matterials to build with. Three swedish mutation mutators.
Rock in shape of a troll face, has invisible BWOINK mines on it.
Asteroid with an abuctor ship crashed into it.
A ship stuffed with cardboard cutouts looking like a swarmer base. It's all fake though.

## Why It's Good For The Game
Mining can get quite bland. More ruins makes it so we can up the spawn rates. 
Give the miners new content!
It also balances in so you can aquire alien tech if you are lucky. Outside of the admin team rolling abductors and being clowned on. 

## Changelog
:cl:
add: A bunch of new mining asteroids!
tweak: Chance to find mining ruins is increased to 33%
tweak: The Aetherwhisp1 AI core asteroid, now has cheesy honkers beneath the AI core.
/:cl:

### CODE REVIEW

**Comment by theboy6545** [comment]
spoodmerge

**Comment by Cenrus** [comment]
Thanks Goat, this PR really rocks.

**Comment by Kmc2000** [comment]
Ram ranch really rocks

File: nsv13/code/modules/mining/asteroid.dm [hunk 1]
```diff
@@ -237,10 +237,10 @@ GLOBAL_LIST_EMPTY(asteroid_spawn_markers)		//handles mining asteroids, kind of s
 	linked.relay('nsv13/sound/effects/ship/tractor.ogg', "<span class='warning'>DANGER: Magnet has locked on to an asteroid. Vacate the asteroid cage immediately.</span>")
 	cooldown = TRUE
 	addtimer(VARSET_CALLBACK(src, cooldown, FALSE), 1 MINUTES)
-	if(prob(10)) //I hate this but it works so fuck you
+	if(prob(33)) //I hate this but it works so fuck you
```
Comment (GoatMonarch) [inline nsv13/code/modules/mining/asteroid.dm:240]: Not entirely sure if this is what I had to change, to increase the ruin chance.
Sorry if I am dumb maintainers.

Comment (Kmc2000) [reply nsv13/code/modules/mining/asteroid.dm:240]: Yep it is, I think

**Review by Vasily2013 — APPROVED** [review]
All looks good to me.

**Pull request merged**